### DISCUSSION TITLE
RTMP Publisher for desktop java application

### DISCUSSION THREAD

**Discussion by fidesetratio** [pedroSG94/RootEncoder#1360] [General]
Guys, can i use Rtmp Publisher from file in java desktop application.
Is There any example that i can use

**Comment by pedroSG94** [comment]
Hello,

No, you can't use RtmpFromFile in a java desktop app because that class has dependencies of Android api.
Anyway, You can use rtmp, rtsp and srt modules doing minimal changes and get video/audio frames yourself
